kolibri/core/assets/src/views/Navbar/NavbarLink.vue

```diff
@@ -131,10 +137,44 @@
 
   .tab-title {
     display: inline-block;
-    font-weight: bold;
     text-overflow: ellipsis;
-    text-transform: uppercase;
-    vertical-align: middle;
+  }
+
+  /deep/ .ui-icon {
+    width: 22px !important;
+    height: 22px !important;
+  }
+
+  /deep/ svg {
+    width: 20px !important;
+    height: 20px !important;
+  }
+
+  @media (max-width: 600px) {
+    .tab {
+      padding: 0 8px;
+      font-size: 12px;
+    }
+
+    .tab-icon {
+      display: inline-block;
+      padding: 10px 0;
+      margin-right: 0;
+    }
+
+    .router-link-active {
+      border-bottom-width: 2px;
+    }
+
+    /deep/ .ui-icon {
+      width: 20px !important;
+      height: 20px !important;
+    }
+
+    /deep/ svg {
+      width: 16px !important;
+      height: 16px !important;
+    }
```

Comment:
Does this target the `.icon` class element? Does this only work if we use `/deep/`?

On a separate note, I'm wondering if that `UiIcon` is necessary at all since we're putting a `KIcon` in there wherever this component is used. Perhaps we could avoid having to `/deep/` anything here if we just put styles directly onto the `KIcon` passed to the slot?

Reply:
this is such a great suggestion - thank you! I've updated